### PR TITLE
#83 Add a command to list registred hooks of a module

### PR DESCRIPTION
**Pull request by nenes25**
Add new functionality from  issue #83 
New command bin/console fop:module:hooks _modulename_

### CODE REVIEW

**Review by SebSept — APPROVED** [review]
Good job :)

File: src/Commands/ModuleHooks.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                $table->render();
+            } else {
+                $io->title('The module is not hooked to any hook');
+            }
+
+            return 1;
+        } else {
+            $io->error('Error the module ' . $moduleName . ' doesn\'t exists');
+
+            return 0;
```
Comment (nenes25) [inline src/Commands/ModuleHooks.php:86]: @SebSept  
Should not this line be replaced by a return 1 ?
And a return 0 outside the else block ?

Comment (SebSept) [reply src/Commands/ModuleHooks.php:86]: Damned, I inverted !
0 is success, other error

**Comment by SebSept** [comment]
@jf-viguier can you just make a test, so we can merge it.

**Review by jf-viguier — APPROVED** [review]
ok for me, I've tested and it's working well

**Pull request merged**